Tutorial/Window Help/wh_shop.csv

```diff
@@ -1,2 +1,2 @@
 name01#0,"""Shops"""
-whelp_shop01#0,"""<yellow>About Shops<c><br><br>At shops, you can buy and sell items. You can also buy back items that you sold by mistake.<br><br>You can purchase items from the Goods List, or sell items from your Inventory, by moving the cursor up and down, changing the number on the item with the left and right arrows, then selecting \""Buy\"" or \""Sell\"".<br><br>[Link to Online Manual]<br><&URL pso2_players_manual_preparations_arkslobby_shoparea>"""
+whelp_shop01#0,"""<yellow>About Shops<c><br><br>At shops, you can buy and sell items. You can also buyback items that you sold by mistake.<br><br>You can purchase items from the Goods List, or sell items from your Inventory, by moving the cursor up and down, changing the number on the item with the left and right arrows, then selecting \""Buy\"" or \""Sell\"".<br><br>[Link to Online Manual]<br><&URL pso2_players_manual_preparations_arkslobby_shoparea>"""
```

Comment:
I think "buy back" is correct here. Buyback is the name of the function, "buy back" is what it does.